feature_layers/display-route-layer/src/main/java/com/esri/samples/display_route_layer/DisplayRouteLayerSample.java

```diff
@@ -0,0 +1,168 @@
+import com.esri.arcgisruntime.layers.FeatureCollectionLayer;
+import com.esri.arcgisruntime.loadable.LoadStatus;
+import com.esri.arcgisruntime.mapping.ArcGISMap;
+import com.esri.arcgisruntime.mapping.BasemapStyle;
+import com.esri.arcgisruntime.mapping.Viewpoint;
+import com.esri.arcgisruntime.mapping.view.MapView;
+import com.esri.arcgisruntime.portal.Portal;
+import com.esri.arcgisruntime.portal.PortalItem;
+import javafx.application.Application;
+import javafx.application.Platform;
+import javafx.geometry.Insets;
+import javafx.geometry.Pos;
+import javafx.scene.Scene;
+import javafx.scene.control.Label;
+import javafx.scene.control.ScrollPane;
+import javafx.scene.layout.*;
+import javafx.scene.paint.Paint;
+import javafx.scene.text.TextAlignment;
+import javafx.stage.Stage;
+
+
+public class DisplayRouteLayerSample extends Application {
+
+    private MapView mapView;
+    private PortalItem portalItem; // keep loadable in scope to avoid garbage collection
+    private FeatureCollection featureCollection;  // keep loadable in scope to avoid garbage collection
+
+
+    @Override
+    public void start(Stage stage) {
+
+        try {
+            // create stack pane and application scene
+            StackPane stackPane = new StackPane();
+            Scene scene = new Scene(stackPane);
+
+            // set title, size, and add scene to stage
+            stage.setTitle("Display Route Layer Sample");
+            stage.setWidth(800);
+            stage.setHeight(700);
+            stage.setScene(scene);
+            stage.show();
+
+            // authentication with an API key or named user is required to access basemap and other location services
+            String yourAPIKey = System.getProperty("apiKey");
+            ArcGISRuntimeEnvironment.setApiKey(yourAPIKey);
+
+            // create a map with the topographical basemap style
+            ArcGISMap map = new ArcGISMap(BasemapStyle.ARCGIS_TOPOGRAPHIC);
+
+            // create a map view and set the map to it
+            mapView = new MapView();
+            mapView.setMap(map);
+
+            // set a viewpoint on the map view
+            mapView.setViewpoint(new Viewpoint(45.2281, -122.8309 + 0.2, 57e4));
+
+            // create a VBox to contain all the UI controls
+            var vBox = new VBox();
+            vBox.setMaxSize(300, 400);
+            vBox.setPadding(new Insets(10.0));
+            vBox.setAlignment(Pos.TOP_CENTER);
+            vBox.setSpacing(10);
+            vBox.setBackground(new Background(new BackgroundFill(Paint.valueOf("rgba(0, 0, 0 , 0.7)"), CornerRadii.EMPTY, Insets.EMPTY)));
+
+            //create a scrollPane to enable scrolling through the directions
+            var scrollPane = new ScrollPane();
+            scrollPane.setMaxSize(300, 400);
+            scrollPane.setPadding(new Insets(10.0));
+
+            // create a VBox to contain the directions
+            VBox directionsVBox = new VBox();
```

Comment:
I think we can use `var` here - it's obvious from both the variable name and constructor that what we're creating is a `VBox`.